suite-common/suite-utils/src/parseFirmwareChangelog.ts

```diff
@@ -0,0 +1,45 @@
+import { FirmwareRelease } from '@trezor/connect';
+
+export type ParseFirmwareChangelogParams = {
+    release?: FirmwareRelease;
+    isBtcOnly: boolean;
+};
+
+export type ParseFirmwareChangelogResult = {
+    url: string;
+    notes: string | undefined;
+    changelog: string;
+    versionString: string;
+};
+
+type ChangelogField = keyof Pick<FirmwareRelease, 'changelog' | 'changelogBtcOnly'>;
```

Comment:
Seems unnecessary to declare, the type is inferred and not used anywhere else.

Reply:
It is declared to be used at line `19` so the strings of the keys of the FirmwareRelease are checked at this line and IDE can autocomplete them.

Without this, typo in those fields would raise error later.

I think having error highlighted by TS at place where it is (at the source) is highly beneficial.